phi/eval/eval.py

```diff
@@ -208,12 +215,59 @@ def print_result(self, answer: Optional[Union[str, Callable]] = None) -> Optiona
             title_style="bold sky_blue1",
             title_justify="center",
         )
-        table.add_row("Question", self.question)
-        table.add_row("Answer", self.answer)
-        table.add_row("Expected Answer", self.expected_answer)
+        table.add_row("Question", question)
+        table.add_row("Answer", answer)
+        table.add_row("Expected Answer", expected_answer)
         table.add_row("Accuracy Score", f"{str(result.accuracy_score)}/10")
         table.add_row("Accuracy Reason", result.accuracy_reason)
+        table.add_row("Relevance Score", f"{str(result.relevance_score)}/10")
+        table.add_row("Completeness Score", f"{str(result.completeness_score)}/10")
+        table.add_row("Confidence Score", f"{str(result.confidence_score)}/1.0")
         table.add_row("Time Taken", f"{response_timer.elapsed:.1f}s")
         console.print(table)
 
         return result
+    
+    def eval_tool_calls(self, expected_tool_calls: List[str], agent: Optional[Agent] = None, prompt: Optional[str] = None, response: Optional[RunResponse] = None) -> bool:
```

Comment:
Should the agent invoke not happen here and you get the response from that? I don't think we need to support both right? Or do we want flexibility so user's can run it themselves and just "assert" ?